working to add ListRetain (#115)
momentohq/public-dev-docs: docs/develop/api-reference/list-collections.md

### ListFetch

Gets a list item from a cache, with optional slices.

| Name       | Type   | Description                                                                 |
|------------|--------|-----------------------------------------------------------------------------|
| cacheName  | String | Name of the cache.                                                          |
| listName   | String | The name of the list item to be retrieved.                                  |
| startIndex | Number | The starting inclusive element of the list to fetch. Default is 0.          |

> | endIndex   | Number | The ending exclusive element of the list to fetch. Default is end of list.  |

Reviewer: do we have any convention for indicating these are optional?  Or should we add (optional) in the description?

Author: Which ones are optional?

Reviewer: startIndex and endIndex

</details>

### ListRetain

> Retains only slice of the list where the start is inclusive and the end is exclusive, what isn't retained, will be dropped from the list.

Reviewer: ```suggestion

Retains only slice of the list where the start is inclusive and the end is exclusive.  The items outside of this range will be dropped from the list.
```
